types/index.d.ts

```diff
@@ -882,14 +883,17 @@ export type ConsumerEachMessagePayload = EachMessagePayload
  */
 export type ConsumerEachBatchPayload = EachBatchPayload
 
+export type EachBatchHandler = (payload: EachBatchPayload) => Promise<void>
+export type EachMessageHandler = (payload: EachMessagePayload) => Promise<void>
```

Comment:
I've explicitly exported these two types to allow using them in utils and libraries (another small pain-point of mine :)